src/test/java/org/openapitools/jackson/nullable/StreamingApiTest.java

```diff
@@ -0,0 +1,421 @@
+/*
```

Comment:
I didn’t notice the copyright in any other files (maybe I missed some).

Are you ok with removing this? I don’t exactly know the difference whether you add this or not though. I _assume_ you’re the copyright holder either way. (…I’m not a lawyer though, so I could be completely wrong.)

Reply:
I can drop this, if it makes things better for the project.

Reply:
Honestly, I just don't know the implications. @wing328 might be able to provide some guidance?